lib/libspl/include/assert.h

```diff
@@ -97,7 +97,7 @@ do {									\
 	const uint64_t __right = (uint64_t)(RIGHT);			\
 	if (!(__left OP __right))					\
 		libspl_assertf(__FILE__, __FUNCTION__, __LINE__,	\
-		    "%s %s %s (0x%llx %s 0x%llx)", #LEFT, #OP, #RIGHT,	\
+		    "%s %s %s (%llx %s %llx)", #LEFT, #OP, #RIGHT,	\
```

Comment:
Why did you remove "0x" here, but left it in above cases?

Reply:
Unintentional, will revert